main.c

```diff
@@ -50,7 +94,7 @@ int main( int argc, char *argv[] )
 	char *physmem = page_table_get_physmem(pt);
 
 	if(!strcmp(program,"sort")) {
-		sort_program(virtmem,npages*PAGE_SIZE);
+		sort_program(virtmem, npages * 5);
```

Comment:
I had the page count set to 5 hear in an earlier commit, but I would set it to PAGE_SIZE when testing so that you know it works.

Reply:
Also, I was testing the paging in the other PRs by printing, in `program.c`, the sum of sorting before and after running the quick-sort. Like https://github.com/elliott-beach/project2_base/blob/59c471a75a3fd18f3acd5385c266b4d527bde8d1/program.c#L63-L82

Reply:
`cur_page++` should go after page is evicted, I think, not after every page.

Even with that change, this doesn't pass that test I suggested, not sure way.

Reply:
I am assuming you are referring to `cur_frame` instead of `curr_page`? If so, I can make that change.